invenio_checks/templates/semantic-ui/invenio_checks/requests/details.html

```diff
@@ -0,0 +1,58 @@
+{# -*- coding: utf-8 -*-
+
+This file is part of Invenio.
+Copyright (C) 2025 CERN.
+
+Invenio is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% import "invenio_checks/requests/severity_level_icons.html" as sli %}
```

Comment:
for code readability we usually prefer explicit variable naming:
```suggestion
{% import "invenio_checks/requests/severity_level_icons.html" as service_level_icons %}
```